subsys/suit/ipuc/src/suit_ipuc_sdfw.c

```diff
@@ -371,6 +362,21 @@ suit_plat_err_t suit_ipuc_sdfw_write(int ipc_client_id, struct zcbor_string *com
 			return SUIT_PLAT_ERR_IO;
 		}
 
+		if ((ipuc_sink.write != NULL) && (offset > ipuc_entry->write_peek_offset)) {
+			/* Erasing if necessary
+			 */
+			uint8_t erase_buffer[READ_CHUNK_MAX_SIZE] = {
+					[0 ... READ_CHUNK_MAX_SIZE - 1] = 0xFF
+			};
+
+			ipuc_sink.seek(ipuc_sink.ctx, ipuc_entry->write_peek_offset);
+			for (size_t i = ipuc_entry->write_peek_offset; i < offset;
```

Comment:
If you find it better - you may replace generic `i` with something like `off` to indicate that it is representing the active offset of the write op.

Reply:
Ok, changed